src/plugins/workspace/public/components/workspace_creator/workspace_form.tsx

```diff
@@ -194,7 +214,7 @@ export const WorkspaceForm = ({
       if (features.length === 0) {
         return previousValue;
       }
-      if (features.length === 1 || currentKey === 'undefined') {
```

Comment:
Remove `features.length === 1` will cause one category with only one feature, this would be a little bit confused. Is this a requirement?